### ISSUE TITLE
[Feature request] Ability to on/off specific shader in preset

### ISSUE THREAD

**Issue by ofry** [libretro/RetroArch#8289]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Ability to on/off specific shader in preset for debugging purposes.

Related to https://github.com/libretro/RetroArch/issues/8176

### Expected behavior

I can independently on/off specific shader in current preset in "Shaders" menu

### Actual behavior

Now I can't do it.

### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: feb6b19

### Environment information

- OS: Windows 8.1 Enterprise 64-bit
- Compiler: [In case you are running local builds]

**Comment by orbea** [comment]
Not a duplicate of issue https://github.com/libretro/RetroArch/issues/6320?

**Comment by ofry** [comment]
No, it isn't.

In this PR I want to on/off any specific shader in preset (ie skip it).

**Comment by orbea** [comment]
I don't understand how this would be used and why a general toggle for shaders is not enough?

**Comment by ofry** [comment]
Let's see this issue in my PC.

If I run retroarch_debug.exe, I got this spam in log:

```
[ERROR] [GL debug (High, API, Error)]: glUniform2fv has failed to set the uniform at location '0': the uniform's type, component size, or matrix dimensions are incompatible with the function variant (GL_INVALID_OPERATION)
```

If I totally disable postprocessing shaders, I got rid of them. So, at least one of shader in my preset is buggy.

This request will help to localize such bugs.

**Comment by orbea** [comment]
Try disabling your shaders one by one until you figure out which one causes that message.

**Comment by ofry** [comment]
@orbea Yes, I should do that. But I want to simplify this operation. So I put this issue.
